### DISCUSSION TITLE
安全问题，脱敏没在后端处理，通过接口能拿到明文

### DISCUSSION THREAD

**Discussion by AllenZou** [arextest/arex-agent-java#520] [Q&A]
<img width="1409" alt="企业微信截图_b3358cee-4b98-4778-a0a0-7e13f9df661e" src="https://github.com/arextest/arex-agent-java/assets/1622669/be2a572b-0070-41ee-9aba-cf215b6cf193">

**Comment by coryhh** [comment]
@AllenZou Thank you for posting in the AREX community! We value your feedback and our team is paying attention to the feedback, feature requests & bugs we are hearing from users in the Community Discussions.

This is a bug, we'll fix this out as soon as we can.
